src/Console/ConsoleOptionParser.php

```diff
@@ -407,6 +414,32 @@ public function epilog($text = null)
         return $this->getEpilog();
     }
 
+    /**
+     * Gets the subcommandSort Property
+     *
+     * @return bool
+     */
+    public function getSubcommandSort()
+    {
+        return $this->_subcommandSort;
+    }
+
+    /**
+     * Sets the subcommandSort property.
+     *
+     * @param bool $sort Whether or not the ConsoleOptionParser should sort subcommands
+     *
+     * @return $this
+     */
+    public function setSubcommandSort($sort)
```

Comment:
In other places we use `enableSubcommandSort()` as a method name.

Reply:
Is this due to the boolean type for $subcommandSort ? Also, since it's set to true by default here, shouldn't it be disableSubcommandSort() as well?

Reply:
you can make it set to True by default.